packages/frontend/src/hooks/echarts/useEcharts.test.ts

```diff
@@ -0,0 +1,126 @@
+import { getAxisDefaultMaxValue, getAxisDefaultMinValue } from './useEcharts';
+
+describe('getAxisDefaultMinValue', () => {
+    test('get axis default min value for NaN', () => {
+        expect(getAxisDefaultMinValue({ min: '', max: 5 })).toBeUndefined();
+        expect(getAxisDefaultMinValue({ min: 10, max: '' })).toBeUndefined();
+        expect(getAxisDefaultMinValue({ min: '', max: '' })).toBeUndefined();
+        expect(
+            getAxisDefaultMinValue({ min: undefined, max: undefined }),
+        ).toBeUndefined();
+        expect(
+            getAxisDefaultMinValue({ min: null, max: null }),
+        ).toBeUndefined();
+        expect(
+            getAxisDefaultMinValue({
+                min: new Date('2021-03-10T00:00:00.000Z'),
+                max: new Date('2021-03-10T00:00:00.100Z'),
+            }),
+        ).toBeUndefined();
+    });
+
+    test('get axis default min value for positive numbers', () => {
```

Comment:
Can we split this test into those we expect to be undefined vs. return a minimum value?